js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -2287,26 +2287,26 @@ createClient=Create client
+fullNameLdapReadOnlyHelp=For Read-only, data is imported from LDAP to Keycloak database, but it is not saved back to LDAP when the user is updated in Keycloak.
 roleExplain=Realm roles are the roles that you define for use in the current realm.
 whatIsDefaultGroups=What is the function of default groups?
 transient=Transient
 generalSettings=General settings
 addClientProfileError=Could not create client profile\: '{{error}}'
 overallResults=Overall Results
-requiredUserActionsHelp=Require an action when the user logs in. 'Verify email' sends an email to the user to verify their email address. 'Update profile' requires user to enter in new personal information. 'Update password' requires user to enter in a new password. 'Configure OTP' requires setup of a mobile password generator.
-requestObjectSignatureAlgorithmHelp=JWA algorithm, which client needs to use when sending OIDC request object specified by 'request' or 'request_uri' parameters. If set to 'any', Request object can be signed by any algorithm (including 'none' ).
+requiredUserActionsHelp=Require an action when the user logs in. 'Verify email' sends an email to the user to verify their email address. 'Update profile' requires user to enter in new personal information. 'Update password' requires the user to enter a new password. 'Configure OTP' requires setup of a mobile password generator.
+requestObjectSignatureAlgorithmHelp=JWA algorithm, which client needs to use when sending OIDC request object specified by 'request' or 'request_uri' parameters. If set to 'any', Request object can be signed by any algorithm (including 'none').
 ldapKerberosSettingsDescription=This section contains options useful for the Kerberos integration. This is used only when the LDAP server is used together with Kerberos/SPNEGO for user authentication.
 deleteEvents=Clear events
 termsOfServiceUrlHelp=URL that the Relying Party Client provides to the End-User to read about the Relying Party's terms of service
 clientSecretError=Could not regenerate client secret due to\: {{error}}
 resourcePath=Resource path
-useJwksUrlHelp=If the switch is on, identity provider public keys will be downloaded from given JWKS URL. This allows great flexibility because new keys will be always re-downloaded again when identity provider generates new keypair. If the switch is off, public key (or certificate) from the Keycloak DB is used, so when the identity provider keypair changes, you always need to import the new key to the Keycloak DB as well.
+useJwksUrlHelp=If the switch is on, identity provider public keys will be downloaded from given JWKS URL. This allows great flexibility because new keys will be always re-downloaded again when identity provider generates new keypair. If the switch is off, public key (or certificate) from the Keycloak database is used, so when the identity provider keypair changes, you always need to import the new key to the Keycloak database as well.
```

Comment:
```suggestion
useJwksUrlHelp=If the switch is on, identity provider public keys are downloaded from a given JWKS URL. The result is great flexibility because new keys are downloaded again when the identity provider generates a new keypair. If the switch is off, a public key (or certificate) from the Keycloak database is used, so when the identity provider keypair changes, you always need to import the new key to the Keycloak database as well.
```